rhino/src/main/java/org/mozilla/javascript/NativeIterator.java

```diff
@@ -249,30 +241,5 @@ public Object __iterator__(boolean b) {
         private Scriptable scope;
     }
 
-    @Override
-    protected int findPrototypeId(String s) {
-        int id;
-        switch (s) {
-            case "constructor":
-                id = Id_constructor;
-                break;
-            case "next":
-                id = Id_next;
-                break;
-            case "__iterator__":
-                id = Id___iterator__;
-                break;
-            default:
-                id = 0;
-                break;
-        }
-        return id;
-    }
-
-    private static final int Id_constructor = 1,
-            Id_next = 2,
-            Id___iterator__ = 3,
-            MAX_PROTOTYPE_ID = 3;
-
     private Object objectIterator;
```

Comment:
Please move the declaration to the top, i think we did that for all other modernized classes